src/interface/src/app/login/login.component.ts

```diff
@@ -1,18 +1,24 @@
-import { Component } from '@angular/core';
+import { Component, Input } from '@angular/core';
 import { FormBuilder, FormGroup, Validators } from '@angular/forms';
 import { Router } from '@angular/router';
 
 import { AuthService } from '../services';
 import { MatDialog } from '@angular/material/dialog';
-import { ResetPasswordDialog } from './reset_password_dialog';
+
+import * as signInMessages from '../shared/constants';
+import { ResetPasswordDialog } from './reset-password-dialog/reset_password_dialog';
 
 @Component({
   selector: 'app-login',
-  templateUrl: './login.component.html',
+  templateUrl: './login.component.ng.html',
```

Comment:
is this file missing from this PR?  I don't see it in the codebase.

Reply:
So I was fixing the naming convention based on a comment made by Maciek and so this comment I think is based on an outdated piece of code.